src/simulators/state_chunk.hpp

```diff
@@ -876,7 +875,10 @@ uint_t StateChunk<state_t>::mapped_index(const uint_t idx)
 template <class state_t>
 void StateChunk<state_t>::apply_chunk_swap(const reg_t &qubits)
 {
-  uint_t q0,q1,nLarge;
+#pragma GCC diagnostic ignored "-Wunused-but-set-variable"
+  uint_t nLarge;
+#pragma GCC diagnostic pop
```

Comment:
Cannot we just initialise `nLarge` to 1? Otherwise, as we support other compilers maybe we would need here some sort of ifdefs to address them as well.

Reply:
That would be much better, I'll update shortly. Thanks